src/communication/messages.py

```diff
@@ -0,0 +1,19 @@
+from enum import Enum
+
+class MessageType(Enum):
+    ERROR = 1
+    OUTPUT_QUEUE_UPDATE = 2
+    STATE_UPDATE = 3
+    MODE_UPDATE = 4
+    BUTTON_CONFIG_UPDATE = 5
+    SONG_UPDATE = 6
+
+class State(Enum):
```

Comment:
Just calling this `State` is a little confuzzling without looking into the code more. A state for what? More descriptive plz